README.md

```diff
@@ -8,18 +8,19 @@ We are one of three development teams in Library IT Reporting to Esme Cowles. Th
 * Research Data and Scholarship Services https://github.com/pulibrary/rdss-handbook
 
 ### Responsibilities 
-The Team is responsible for the business systems, web sites, discovery and data management for the library. We all work remotely at least two days per week. We have daily check-ins to stay connected, with a longer check-in to plan our work for the week. Although, because we are customer focused our priorities may shift throughout the week. We have two open positions within the team. We also have regular standing check-ins with other teams throughout the library.
+The Team is responsible for the discovery systems, web sites, business applications and data management for the library. We all work remotely at least two days per week. We have daily check-ins to stay connected, with a longer check-in to plan our work for the week. Although, because we are customer focused our priorities may shift throughout the week. We have two open positions within the team. We also have regular standing check-ins with other teams throughout the library.
 
-The Team collaborates closely with the three staff members (also currently reporting to Kevin) who manage our Alma platform to support the integrations (indexing, availability, requesting) with the Catalog, as well as supporting the code that facilitates the movement data in and out of Alma related to patron and financial information.
+The Team collaborates closely with the three staff members (also currently reporting to team's manager) who manage our Alma platform to support the integrations (indexing, availability, requesting) with the Catalog, as well as supporting the code that facilitates the movement data in and out of Alma related to patron and financial information.
```

Comment:
Can we change this to `three staff members in Catalog and Metdata Services (CAMS)`